src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -72,19 +71,22 @@ class PhoneEditModal extends React.Component {
     let defaultFieldValue;
 
     if (this.props.data) {
-      defaultFieldValue = merge(this.props.data, {
-        inputPhoneNumber:
-          this.props.data &&
-          [this.props.data.areaCode, this.props.data.phoneNumber].join(''),
+      defaultFieldValue = {
+        ...this.props.data,
+        inputPhoneNumber: `${this.props.data.areaCode}${
+          this.props.data.phoneNumber
+        }`,
+        extension: this.props.data.extension || '',
         'view:showSMSCheckbox': this.props.showSMSCheckbox,
-      });
+      };
     } else {
       defaultFieldValue = {
         countryCode: '1',
         extension: '',
         inputPhoneNumber: '',
         isTextable: false,
         isTextPermitted: false,
+        'view:showSMSCheckbox': this.props.showSMSCheckbox,
```

Comment:
I also noticed that this hadn't been set before, so Vets who were adding a mobile number for the first time who would e eligible for SMS notifications would not even seen the SMS checkbox on the edit modal.

Reply:
:grimacing:

Reply:
But to be clear, this is _not_ on prod yet so it wasn't actually affecting anyone!